src/main/scala-2.10/com/mpc/scalats/core/ScalaParser.scala

```diff
@@ -230,6 +245,28 @@ final class ScalaParser(logger: Logger) {
   @inline private def isAnyValChild(scalaType: Type): Boolean =
     scalaType <:< typeOf[AnyVal]
 
+  @inline private def isEnumerationValue(scalaType: Type): Boolean = {
+    scalaType.typeSymbol.asClass.fullName == "scala.Enumeration.Value"
```

Comment:
Rather compare `Type` (than string)

Reply:
Unfortunately couldn't get this to work any other way... in particular nothing like `scalaType <:< typeOf[scala.Enumeration#Value]` worked. Let me leave a FIXME...